### PR TITLE
XAUTOCLAIM docs

### PR DESCRIPTION
**Pull request by guybe7**

### CODE REVIEW

File: commands.json [hunk 1]
```diff
@@ -5104,6 +5104,39 @@
...
+        "type": "string"
+      },
+      {
+        "name": "start",
+        "type": "string"
+      },
+      {
+        "command": "COUNT",
+        "name": "count",
+        "type": "integer"
```
Comment (oranagra) [inline commands.json:5134]:
```suggestion
        "type": "integer"
        "optional": true
```

**Review by oranagra — COMMENTED** [review]
not sure why the test fails, please look into that.
everything else looks good to me..
@itamarhaber ?

**Comment by itamarhaber** [comment]
@guybe7 please review and tell me if it makes sense

**Comment by oranagra** [comment]
@guybe7 @itamarhaber i pushed two commits.
first one is a refactoring to comply with the future #1485 (so that the diff in the second one is smaller)
second one is the new features, please review.

**Review by itamarhaber — APPROVED** [review]
Great stuff :)

**Pull request merged**